### DISCUSSION TITLE
Bl touch x,y offset

### DISCUSSION THREAD

**Discussion by DrGonzzzo** [Jyers/Marlin#96] [Q&A] [answered]
The X and Y offset in advanced menu within the precompiled fw....Does this apply to where the bl touch is or the nozzle? Because when I shift so that the bl touch probes in the correct places when I attempt to start a print the nozzle is shifted pretty far right and in turn cost me a bit of build area. In which the printer cuts out that part of a print if it happens to go past that line. I'm using this along with my micro Swiss direct drive https://www.thingiverse.com/thing:4548894

**Comment by Deletedice** [comment, marked as answer]
If you have the Bltouch installed then it is where the probe is located. The nozzle will be in the correct place once printing begins. 

See also https://github.com/Jyers/Marlin/discussions/76

**Comment by darco415** [comment]
what are your xy offset settings?

**Reply by DrGonzzzo** [reply]
Ive tried roughly -48, -9 and even the default  -10,10 and either way it's not where it needs to be when I print

**Reply by Deletedice** [reply]
> Ive tried roughly -48, -9 and even the default -10,10 and either way it's not where it needs to be when I print

@DrGonzzzo Have you tried offsets x: -53.1 and y: -18.4 as mentioned in the thingiverse page?

**Reply by DrGonzzzo** [reply]
I don't think the precompiled goes over -50

**Reply by DrGonzzzo** [reply]
And I don't recall seeing those numbers

**Reply by Jyers** [reply]
> I don't think the precompiled goes over -50

Well thats an issue, isn't it... I'll change the limits to -100, 100 in the next release
